I added your commits

ethanetExplorer opened the pull request.

ethanetExplorer committed:
Added $ (dollar sign) in PersonTransactionRow view
It is not a binding variable, just a "$" string

ethanetExplorer committed:
Re-added notifcation toggle in NewTransactionSheet

ethanetExplorer committed:
Removed date ranges

ethanetExplorer committed:
Fixed notifications and added a new dedicated file for transactions, manually added Yuhan's commit 5addfad

ethanetExplorer committed:
Changed display name to Money Rush

ethanetExplorer committed:
Made completed transactions in HomeView appear gray and moved DecimalTextField to New Transaction folder

ethanetExplorer committed:
Final pull request

NewTransactionSheet: Added cancel and save buttons, added changes as found on Yuhan's commit
Every issue I found has been fixed
Yay

TwelvePenguins reviewed (approved):
'APPROVE' performed by Xcode.

ethanetExplorer committed:
Update NewTransactionSheet.swift

Made everything work properly: When transaction is a synchronised bill split, money each and due dates are now the same for everyone in the transaction. Conversely, when transaction is not a synchronised bill split, money and due dates are now different for everyone

ethanetExplorer committed:
Update TransactionRowView.swift

Fixed showing paid people in HomeView instead of unpaid people
